EDU-17996 - [LOC REVIEW] Suporte a contas GitHub Enterprise no WebOps (#2579)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/2-starting-the-project.mdx

Reviewer: Não há headings para essas âncoras. Sugiro linkar a doc do GitHub ou remover.

Obs.: Essa sugestão está fora do diff hunk, então, se usar ela, validar se o texto foi alterado corretamente.
```suggestion
* [GitHub personal account](https://docs.github.com/en/account-and-profile/how-tos) or [organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations)
* [GitHub Enterprise](https://docs.github.com/en/enterprise-cloud@latest/get-started/onboarding/getting-started-with-github-enterprise-cloud)
```

Reviewer: Obs.: Essa sugestão está fora do diff hunk, então, se usar ela, validar se o texto foi alterado corretamente.
```suggestion
> ⚠️ If you choose to install the app in a GitHub organization or GitHub Enterprise, make sure you have [admin access](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).
```

---
title: '2. Starting a new FastStore project'
excerpt: "Create and set up your FastStore project with WebOps."

> updatedAt: "2026-05-15T20:19:39.310Z"

Author: ```suggestion
updatedAt: "2026-05-18T12:45:56.318Z"
```

### Check permissions

> To start a FastStore project, you must have the **Owner (Admin Super)** [role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#roles).

Reviewer: Atualizar link para não precisar de redirect.
```suggestion
To start a FastStore project, you must have the **Owner (Admin Super)** [role](https://help.vtex.com/en/docs/tutorials/roles#roles).
```

### Create your store catalog

Create a catalog to populate and test your store. It should include essential components such as product listing pages, search, product details, and checkout.
Developing without a catalog may result in incomplete testing, data integration issues, and rework later.

> For more information, see the [Catalog documentation](https://help.vtex.com/docs/tracks/catalog).

Author: ```suggestion
For more information, see the [Catalog documentation](https://help.vtex.com/en/docs/tracks/catalog-101).
```
